### DISCUSSION TITLE
Day 5!

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#6] [Ideas]
🎄🎄 **Welcome to Day 5** 🎄🎄

I've seen some great coding solutions so far. Shout out to @Sowmiya07 @em-baggie @hollygrimm for keeping up to date. I will admit I am now one day behind 🫣 

How are you all finding it? What have you learnt so far? What is the most difficult day?

If you are only just joining us, come on in and get stuck in. We are all working at different paces, time zones, etc. but all welcome. 

Cheers
Caroline

**Comment by hollygrimm** [comment]
Each day has been harder, but it's been forcing me to level up in my coding practice, applying the different data structures (ndarray, hashmap, vectors) and ways of interacting with them (regex, windows, loops, maps). Thanks for everybody's hard work!
[My solution for Day 5](https://github.com/hollygrimm/AdventOfCode_WIR/tree/main/day_05)

**Reply by CarolineMorton** [reply]
[This](https://github.com/hollygrimm/AdventOfCode_WIR/blob/main/day_05/src/errors.rs) is really impresssive

**Comment by AR2202** [comment]
Did [day5 in Rust](https://github.com/AR2202/AdventOfCode_WIR/blob/main/day_05/src/main.rs) but it became a bit of a mess.

**Comment by em-baggie** [comment]
<a href="https://github.com/em-baggie/AdventOfCode_WIR/tree/main/day_05">My solution</a> for day 5 part 1. Found this easier than day 4 part 1. Will come back to part 2 if I manage to catch up with the days! Tried again to use custom error handling.

**Comment by kuhmil** [comment]
Was able to sort the safety manuals, here is my version of [Day 5](https://github.com/kuhmil/AdventOfCode_WIR/tree/main/day_05)! Still working on adding tests to this as part of my effort to improve my testing skills in Rust.

**Reply by CarolineMorton** [reply]
Great to see!

**Comment by DominikStefancik** [comment]
My [solution](https://github.com/DominikStefancik/AdventOfCode_WIR/tree/main/day_05) for the Day 5! Added more tests to be sure each step is working correctly.

**Comment by Sowmiya07** [comment]
[Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_05/src/main.rs) is Day 5 Part 1 solution

**Reply by CarolineMorton** [reply]
Awesome to see!
